<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPassword</class>
 <widget class="QWidget" name="ResetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/panj.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>new password :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/sade.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/panj.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>confirm :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/sade.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/panj.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/panj.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>user name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/sade.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/panj.jpg);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
